### ISSUE TITLE
(helm) can't configure APP_URL to use https

### ISSUE THREAD

**Issue by sleeyax** [invoiceninja/dockerfiles#343]
Currently the `APP_URL` config value assumes the configured host communicates over plain HTTP:

https://github.com/invoiceninja/dockerfiles/blob/8e9e7c47da71e2979c214777020cf0e3f3c8c4b7/charts/invoiceninja/templates/configmap.yaml#L15-L19

Because of security reasons I'd prefer to use HTTPS though, so are there any plans to support this?

Maybe allow the programmer to configure the full URL instead (and not just the host)? For example:
```yaml
data:
  {{- if .Values.nginx.ingress.enabled }}
  APP_URL: "{{ .Values.nginx.ingress.url}}"
  {{- else }}
  # ...
```

**Comment by sleeyax** [comment]
To clarify my setup: secure ingress (sub)domains are enforced by my provider. This means I can't access the 'setup' page on initial install over insecure HTTP. When I try to access the page using HTTPS, JS and CSS resources don't load because they are blocked by the browser: `Blocked loading mixed active content “http://example.com/css/app.css”`

As a work-around, I finished the installation by port forwarding the service to my local computer.

**Comment by sleeyax** [comment]
Closing this issue because it's solved and probably a specific edge case most users won't have issues with.

**Comment by sleeyax** [comment]
Ouch, 4 years ago... IIRC the issue solved itself after I updated to a newer version back then.

It's been a long time ago though and eventually, after evaluating invoice ninja extensively, I decided to switch to the paid managed invoice ninja and honestly I couldn't be happier as updating and maintaining the self hosted version gets painful after a while.

**Comment by sleeyax** [comment]
I found this in my old k8s configuration notes:

> Edit the configmap and set APP_URL to use HTTPS (it's HTTP by default for some reason) kubectl -n invoiceninja edit configmap invoiceninja. If you haven't configured the app to force HTTPS yet, also set REQUIRE_HTTPS to true. Restart the invoiceninja deployment afterwards and clear laravel cache if necessary.

Maybe it helps.

**Comment by cianiandreadev** [comment]
Hi @sleeyax ,
Thanks so much for the support!
Yes, I saw somewhere that and indeed I tried a similar approach, but didn't work.
Now I am trying to open a new issue (since I cannot reopen this) and I'll link that to this. 
Hope someone from invoiceninja will pick up this problem finally.
Thanks a lot
